Documentation/troubleshooting.rst

```diff
@@ -31,15 +31,21 @@ fails between endpoints across multiple nods.
 Troubleshooting steps:
 ~~~~~~~~~~~~~~~~~~~~~~
 
-1. Run `cilium monitor` on the node of the source and destination endpoint.
+1. Run `cilium-health status` on the node of the source and destination
+   endpoint. It should describe the connectivity from that node to other
+   nodes in the cluster, and to a simulated endpoint on each other node. If
+   the command does not describe the status of the other node, there may be an
+   issue with the KV-Store.
```

Comment:
In line with `Look for packet drops` below, what should the troubleshooter look for here? I'm not sure how much we should add, though, since this "step" is already multiple lines long.

Reply:
Hehe, when you put it that way it is rather obvious. Staring at it for a bit I realised what I was missing. I can look at the `cilium-health status` output and guess what an error on each line means, but I couldn't trivially find an explanation of what each one maps to (does an ICMP error mean a policy problem or that the nodes can't talk on the network, what about the HTTP via L3 case).
You shouldn't block on this for this PR because I'm holding you up needlessly, but a section with a table explaining some causes for each failure mode reported by cilium-health, linked to from here, would work for me. I'm also happy to take a stab at this as a test of what I understand of cilium-health :)